pkg/flag/report_flags.go

```diff
@@ -216,19 +227,28 @@ func (f *ReportFlagGroup) ToOptions() (ReportOptions, error) {
 		return ReportOptions{}, xerrors.Errorf("unable to load compliance spec: %w", err)
 	}
 
+	var outputPluginArgs []string
+	if arg := getString(f.OutputPluginArg); arg != "" {
+		outputPluginArgs, err = shellwords.Parse(arg)
```

Comment:
Can we add small test to see that this function works as expected?

Reply:
Added in https://github.com/aquasecurity/trivy/pull/4863/commits/43d71ffdda967a53fa1fd6a001d0ca4f39a0703e